charts/console/README.md

```diff
@@ -2,7 +2,7 @@
 
 # Conduktor Console
 
-> If you have any questions or [feedback](https://product.conduktor.help/c/55-helm-chart) contact our [support](https://www.conduktor.io/contact/support/).
+> If you have any questions you can [submit feedback](https://support.conduktor.io/hc/en-gb/requests/new?ticket_form_id=17438365654417) or contact [support](https://www.conduktor.io/contact/support/).
```

Comment:
conduktor.io/roadmap

Reply:
The above one works unless want to change again? (already merged)
Is conduktor.io/roadmap DNS to the form?